lib/common/diagramvisitor.js

```diff
@@ -247,6 +265,16 @@ class DiagramVisitor {
             parameters.fileWriter.writeLine(0, `${source} ${DiagramVisitor.INHERITANCE} ${target}`);
         }
     }
+
+    /**
+     * Visitor design pattern
+     * @param {Decorator[]} decorators - the objects being visited
+     * @param {Object} parameters  - the parameter
+     * @protected
+     */
+    handleDecorators(decorators, parameters) {
+        decorators.forEach(decorator => decorator.accept(this, parameters));
```

Comment:
Let's hoist this to each visit method. I know that it's a little redundant, but it follows the visitor pattern better.

Reply:
Alright